core/templates/dev/head/pages/exploration_editor/editor_tab/StateResponses.js

```diff
@@ -17,346 +17,16 @@
-          // cover most cases.
-          //
-          // TODO(sll): Find a way to make this fully deterministic. This can
-          // probably only occur after we support custom editors for
-          // interactions.
-          var onlyEditsHappened = false;
-          if (oldAnswerChoices.length === newAnswerChoices.length) {
-            onlyEditsHappened = true;
-
-            // Check that no answer choice appears to have been moved.
-            var numAnswerChoices = oldAnswerChoices.length;
-            for (var i = 0; i < numAnswerChoices; i++) {
-              for (var j = 0; j < numAnswerChoices; j++) {
-                if (i !== j &&
-                    oldAnswerChoices[i].val === newAnswerChoices[j].val) {
-                  onlyEditsHappened = false;
-                  break;
-                }
-              }
-            }
-          }
-
-          var oldChoiceStrings = oldAnswerChoices.map(function(choice) {
-            return choice.val;
-          });
-          var newChoiceStrings = newAnswerChoices.map(function(choice) {
-            return choice.val;
-          });
-
-          _answerGroups.forEach(function(answerGroup, answerGroupIndex) {
-            var newRules = angular.copy(answerGroup.rules);
-            newRules.forEach(function(rule) {
-              for (var key in rule.inputs) {
-                var newInputValue = [];
-                rule.inputs[key].forEach(function(item) {
-                  var newIndex = newChoiceStrings.indexOf(item);
-                  if (newIndex !== -1) {
-                    newInputValue.push(item);
-                  } else if (onlyEditsHappened) {
-                    var oldIndex = oldChoiceStrings.indexOf(item);
-                    if (oldIndex !== -1) {
-                      newInputValue.push(newAnswerChoices[oldIndex].val);
-                    }
-                  }
-                });
-                rule.inputs[key] = newInputValue;
-              }
-            });
-
-            _updateAnswerGroup(answerGroupIndex, {
-              rules: newRules
-            });
-          });
-        }
-      },
-      getAnswerGroups: function() {
-        return angular.copy(_answerGroups);
-      },
-      getAnswerGroup: function(index) {
-        return angular.copy(_answerGroups[index]);
-      },
-      getAnswerGroupCount: function() {
-        return _answerGroups.length;
-      },
-      getDefaultOutcome: function() {
-        return angular.copy(_defaultOutcome);
-      },
-      getConfirmedUnclassifiedAnswers: function() {
-        return angular.copy(_confirmedUnclassifiedAnswers);
-      },
-      // This registers the change to the handlers in the list of changes, and
-      // also updates the states object in explorationStatesService.
-      save: function(newAnswerGroups, defaultOutcome) {
-        _saveAnswerGroups(newAnswerGroups);
-        _saveDefaultOutcome(defaultOutcome);
-      }
-    };
-  }
-]);
 
```

Comment:
Please remove this extra line.

Reply:
There are two new lines (line 19 and 20) in StateResponse.js, right? So we can just make it one newline.